### ISSUE TITLE
Issue with comparison with cleverhans 

### ISSUE THREAD

**Issue by neale** [bethgelab/foolbox#108]
I've been using foolbox for quite some time and to be thorough I've been doing comparisons with cleverhans, since that's what a lot of people in the community are going to be testing with. 

I've noticed an issue, particularly with FGSM where cleverhans is somehow mounting a much stronger attack. On reviewing both implementations I can't find a suitable difference. 

If I load identical models and run seemingly identical FGSM attack scripts on MNIST with an epsilon of 0.3. 
I get these results:
```
test_foolbox.py
Clean accuracy: 0.998%
Attack success: 0.477%
Average Linf norm: 0.300000011921
```

```
test_tf.py
Clean accuracy: 0.9980
Adversarial accuracy: 0.2160
```

I created a test repo with two files, one for cleverhans, and one for foolbox, using the same keras model def and weights [here]. (https://github.com/neale/fgsm_test) The cleverhans test script is just a shortened version of one of their [tutorials](https://github.com/tensorflow/cleverhans/blob/master/cleverhans_tutorials/mnist_tutorial_keras_tf.py) without the adversarial training. ,

**Comment by jonasrauber** [comment]
Hi @neale, thanks for reporting this. This was indeed a numerical issue that only happens for Keras models that provide probabilities instead of logits as output. I just released version 0.14.0 that fixes this bug. With that, the Foolbox FGSM attack now achieves `Attack success: 0.786` which corresponds to the `1 - 0.2160 = 0.784` of CleverHans.

Can you confirm these results? If so, please close the issue. Thanks!
